packages/ifpack2/src/Ifpack2_BlockTriDiContainer_impl.hpp

```diff
@@ -3842,11 +3849,14 @@ namespace Ifpack2 {
 
               // y := x - R y
               Z.doImport(Y, *tpetra_importer, Tpetra::REPLACE);
```

Comment:
Is doImport the trouble guy using a host local view ? At my glance, I do not use Tpetra routine in this function except for that. If so, the code path is for testing only. We can probably create a workaround and keep the code as it is. For instance, we can enable the sequential path in this and testing code when UVM is enabled only.

Reply:
@kyungjoo-kim Yes it is - doImport gets a ReadWrite hostview of Z and that was causing the exception.